src/firebase/actionInDatabase/deleteUser.ts

```diff
@@ -0,0 +1,8 @@
+import { deleteDoc, doc } from 'firebase/firestore'
+import { firebaseCollection } from '../../types'
+import { db } from '../firebase'
+
+export const deleteUser = async (userUUID: string) => {
+    await deleteDoc(doc(db, firebaseCollection.users, userUUID))
```

Comment:
aa, either
- you return it and remove await
- remove async/await - it will work the same